util/java/libtiled-java/src/main/java/org/mapeditor/io/TMXMapReader.java

```diff
@@ -83,18 +87,39 @@
     public static final long FLIPPED_VERTICALLY_FLAG = 0xFFFFFFFF40000000L;
     public static final long FLIPPED_DIAGONALLY_FLAG = 0xFFFFFFFF20000000L;
 
-    public static final long ALL_FLAGS = FLIPPED_HORIZONTALLY_FLAG
-            | FLIPPED_VERTICALLY_FLAG
-            | FLIPPED_DIAGONALLY_FLAG;
+    public static final long ALL_FLAGS =
+        FLIPPED_HORIZONTALLY_FLAG | FLIPPED_VERTICALLY_FLAG | FLIPPED_DIAGONALLY_FLAG;
+
+    /**
+     * Set of classes to be bound to the {@link JAXBContext}.
+     * @see #unmarshaller
+     */
+    private static final Set<Class<?>> classesToBeBound = Collections.synchronizedSet(new HashSet<>());
```

Comment:
We can now also avoid the set entirely, right? Now we create set from array and then array again in constructor. Or the list of classes could even be inlined in the constructor.

Reply:
yep